core/workflow-activiti/src/main/resources/userWorkflow.bpmn20.xml

```diff
@@ -82,12 +82,47 @@ under the License.
     
     <serviceTask id="delete" name="Delete" activiti:expression="#{delete.execute(execution.processInstanceId)}"/>
     <sequenceFlow id="flow99" sourceRef="delete" targetRef="theEnd"/>
+    <!-- Recertification tasks -->
+    <userTask id="RecertificationRequest" name="Recertification Request" activiti:candidateGroups="managingDirector" activiti:formKey="recertify">
```

Comment:
Removing some production workflow looks fine, thanks.

You might consider writing some sections about identity recertification in the [reference guide](https://ci.apache.org/projects/syncope/reference-guide.html), also including some operational notes as how to set assignment.